package.json

```diff
@@ -5,7 +5,7 @@
   "main": "discordas.js",
   "scripts": {
     "test": "npm run-script build && mocha build/test/config.js build/test",
-    "lint": "tslint --project ./tsconfig.json -t stylish",
+    "lint": "eslint -c .eslintrc --max-warnings 0 src/**/*.ts test/**/*.ts",
```

Comment:
Why was this set to allow 0 warnings? This fails the CI in my branch #637.
Removing all the warnings is unrealistic with the current ESLint config and code.
Furthermore, warnings would behave just like errors.